src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        final int arrLen = 6;
+
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[arrLen];
+        for (int i = 0; i < arrLen; i++) {
+            if (i < 3) {
```

Comment:
for more consistency 
```suggestion
            if (i < figures.length/2) {
```